sdk/cosmos/benchmark/pom.xml

```diff
@@ -31,24 +10,113 @@
     <artifactId>azure-cosmos-parent</artifactId>
     <version>3.0.0</version>
   </parent>
-  
+
+  <groupId>com.microsoft.azure</groupId>
   <artifactId>azure-cosmos-benchmark</artifactId>
-  <name>Async SDK for SQL API of Azure Cosmos DB Service - Benchmarking tool</name>
-  <description>Benchmarking tool for Async SDK for SQL API of Azure Cosmos DB Service</description>
 
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-  </properties>
+  <name>Microsoft Azure SDK for SQL API of Azure Cosmos DB Service - Benchmarking tool</name>
+  <description>This package contains Benchmarking tool for Microsoft Azure SDK for SQL API of Azure Cosmos DB Service</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-cosmos</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.beust</groupId>
+      <artifactId>jcommander</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-jvm</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-graphite</artifactId>
+    </dependency>
+
+    <!--<dependency>-->
+    <!--<groupId>io.netty</groupId>-->
+    <!--<artifactId>netty-tcnative</artifactId>-->
+    <!--<version>${netty-tcnative.version}</version>-->
+    <!--<classifier>linux-x86_64</classifier>-->
+    <!--</dependency>-->
```

Comment:
we use netty-tcnative for bechmarking on linux.
why is this disabled?

Reply:
added it back.